packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -471,6 +483,33 @@ function getLowestRank(
   return lowestRanks.map(rank => rank.replace(/-/g, ' ')).join(', ');
 }
 
+/**
+ * An enum with specified values would be something like the following:
+ *
+ * ```ts
+ * enum Foo {
+ *   Bar = 111,
+ *   Baz = 222,
+ * }
+ * ```
+ *
+ * An enum without specified values would be something like the following:
+ *
+ * ```ts
+ * enum Foo {
+ *   Bar,
+ *   Baz,
+ * }
+ * ```
+ */
+function isEnumMemberWithSpecifiedValue(member: Member): boolean {
+  if (member.type !== AST_NODE_TYPES.TSEnumMember) {
+    return false;
+  }
+
+  return member.initializer !== undefined;
```

Comment:
Teeny nit: you could make this just a bit more concise...

```suggestion
  return member.type === AST_NODE_TYPES.TSEnumMember && member.initializer !== undefined;
```